django/db/backends/sqlite3/base.py

```diff
@@ -85,6 +85,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     data_types = {
         'AutoField': 'integer',
         'BigAutoField': 'integer',
+        'SmallAutoField': 'integer',
```

Comment:
My understanding is that `smallint AUTOINCREMENT` is not possible. Am I right?

Reply:
Correct. SQLite is weird. All integer types in memory are 64-bit values, except the storage type on disk. It gets even more funky for integer primary keys which essentially alias the hidden `rowid`.

https://www.sqlite.org/datatype3.html
https://www.sqlite.org/lang_createtable.html#rowid